hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamerSchemaEvolutionQuick.java

```diff
@@ -97,19 +106,27 @@ protected static Stream<Arguments> testReorderedColumn() {
   }
 
   protected static Stream<Arguments> testParamsWithSchemaTransformer() {
+    boolean fullTest = false;
     Stream.Builder<Arguments> b = Stream.builder();
-    for (Boolean useTransformer : new Boolean[] {false, true}) {
-      for (Boolean setSchema : new Boolean[] {false, true}) {
-        for (Boolean rowWriterEnable : new Boolean[] {true}) {
-          for (Boolean nullForDeletedCols : new Boolean[] {false, true}) {
-            for (Boolean useKafkaSource : new Boolean[] {false, true}) {
-              for (String tableType : new String[] {"COPY_ON_WRITE", "MERGE_ON_READ"}) {
-                b.add(Arguments.of(tableType, rowWriterEnable, useKafkaSource, nullForDeletedCols, useTransformer, setSchema));
+    if (fullTest) {
+      for (Boolean useTransformer : new Boolean[] {false, true}) {
+        for (Boolean setSchema : new Boolean[] {false, true}) {
+          for (Boolean rowWriterEnable : new Boolean[] {true}) {
+            for (Boolean nullForDeletedCols : new Boolean[] {false, true}) {
+              for (Boolean useKafkaSource : new Boolean[] {false, true}) {
+                for (String tableType : new String[] {"COPY_ON_WRITE", "MERGE_ON_READ"}) {
+                  b.add(Arguments.of(tableType, rowWriterEnable, useKafkaSource, nullForDeletedCols, useTransformer, setSchema));
+                }
               }
             }
           }
         }
       }
+    } else {
+      b.add(Arguments.of("COPY_ON_WRITE", true, true, true, true, true));
```

Comment:
similar. could we just come up with the "superset"/higher complexity combination here. or is that it?